src/components/EpisodesNav/styles.scss

```diff
@@ -1,14 +1,120 @@
-.paginationLink {
-	font-size: 1em; //placeholder to have the classname showing in inspector
+.wrapper {
+	position: fixed; //with absolute it will scroll with page
+	width: 80%;
+	left: 50%;
+	bottom: 5%;
+	transform: translateX(-50%);
 }
 
-.navButton {
-	width: 50px;	//placeholder so they are visible
-	height: 50px;
-	margin: 5px;
+	.container {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+	}
+
+		.button {
+			font-size: 1.75rem;
+			width: 1em;	
+			height: 1.42em;
+
+			&[aria-hidden="true"] {
+				pointer-events: none; //disable and hide hidden
+				opacity: 0;
+			}
+		}
+
+		.prev {
+			order: 1;
+		}
+
+		.next {
+			order: 3;
+		}
+
+		.pagination {
+			order: 2;
+			display: flex;
+			width: 40%;
+			height: 1.5em;
+			align-items: center;
+			list-style: none;
+		}
+
+			.inactive, .active {
+				display: flex;
+				position: absolute;
+				height: 100%;
+				left: 50%;
+				align-items: center; 
+
+				a {
+					text-decoration: none;
+				}
+			} 
+
+				.inactive {
+					font-size: 1.15em;
+					font-weight: normal;
+					pointer-events: none; //prevents overlap with prev/next
+				}
+
+				.active {
+					font-size: 1.5em;
+					font-weight: bold;
+					pointer-events: all;
+				}
```

Comment:
We can do `&.inactive` and `&.active` and put them as a child of `.item` now that we have that rest state in!